py4cast/plots.py

```diff
@@ -489,7 +488,7 @@ class StateErrorPlot(Plotter):
 
     def __init__(
         self,
-        metrics: Dict[str, Py4CastLoss],
+        metrics: Dict[str, Any],
```

Comment:
could we not specify a bit more on this type ? like either iterable, tensor or array ?

Reply:
Done